packages/mdc-textfield/foundation.js

```diff
@@ -98,12 +127,21 @@ export default class MDCTextfieldFoundation extends MDCFoundation {
     this.adapter_.removeHelptextAttr(ARIA_HIDDEN);
   }
 
+  transitionEnd_(evt) {
+    const {BOTTOM_LINE_ACTIVE} = MDCTextfieldFoundation.cssClasses;
+
+    if (evt.propertyName === 'opacity' && !this.isFocused_) {
```

Comment:
We need the line to be completely transparent before removing the `*__bottom-line--active` class, otherwise we see the line start to shrink when an input is blurred.